src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +14,66 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't save a Comment entity with ID:" + entity.getId()
+                    + "\t Exception: ", e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+        return entity;
     }
 
     @Override
     public Comment get(Long id) {
-        return null;
+        Session session = factory.openSession();
+        Comment entity = session.get(Comment.class, id);
+        session.close();
+        return entity;
     }
 
     @Override
     public List<Comment> getAll() {
-        return null;
+        Session session = factory.openSession();
+        List<Comment> entityList =
+                session.createQuery("SELECT a FROM Comment a", Comment.class).getResultList();
```

Comment:
```suggestion
                session.createQuery("FROM Comment", Comment.class).getResultList();
```

you can simplify this